src/test/fuzz/mini_miner.cpp

```diff
@@ -133,12 +133,12 @@ FUZZ_TARGET(mini_miner_selection, .init = initialize_miner)
     const auto block_adjusted_max_weight = MAX_BLOCK_WEIGHT - MINIMUM_BLOCK_RESERVED_WEIGHT;
 
     LOCK2(::cs_main, pool.cs);
-    LIMITED_WHILE(fuzzed_data_provider.ConsumeBool(), 100)
+    LIMITED_WHILE(fuzzed_data_provider.ConsumeBool(), 500)
```

Comment:
Worth noting this is pegged to the number at which `GatherClusters` halts (which would lead to `!IsReadyToCalculate`)

Reply:
Yepp, I actually ran into this during my first iteration of working on this. I will add a comment to explain it.